service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/mappers/StønadsstatistikkMapper.kt

```diff
@@ -122,7 +122,7 @@ private fun vedtakstype(vedtak: VedtakSomKanRevurderes.EndringIYtelse) = when (v
     is VedtakSomKanRevurderes.EndringIYtelse.InnvilgetSøknadsbehandling -> Statistikk.Stønad.Vedtakstype.SØKNAD
     is VedtakSomKanRevurderes.EndringIYtelse.OpphørtRevurdering -> Statistikk.Stønad.Vedtakstype.REVURDERING
     is VedtakSomKanRevurderes.EndringIYtelse.StansAvYtelse -> Statistikk.Stønad.Vedtakstype.STANS
-    is VedtakSomKanRevurderes.EndringIYtelse.InnvilgetRegulering -> TODO()
+    is VedtakSomKanRevurderes.EndringIYtelse.InnvilgetRegulering -> Statistikk.Stønad.Vedtakstype.REGULERING
```

Comment:
Har vi sjekket med statistikkgjengen og blitt enige om disse verdiene?

Reply:
Nei, er nog lurt å dobbelsjekke med de ja. Slenger iväg en melding.

Reply:
Tror det er et fett for de om vi kaller det regulering eller "behandling type 3", så bare kjør på. Tror de heller ikke er i prod med dette, så kan bare la de evt. si ifra til oss i test hvis de vil ha ting navngitt annerledes.